locales/de/LC_MESSAGES/volto.po

```diff
@@ -26,6 +26,14 @@ msgstr ""
 "Preferred-Encodings: utf-8 latin1\n"
 "X-Is-Fallback-For: de-at de-li de-lu de-ch de-de\n"
 
+#: components/manage/Controlpanels/UsersControlpanel
+msgid "ADD NEW GROUP"
```

Comment:
@nileshgulia1 the message string should not be all upper-case. You can make the string appear all upper-case with CSS.